src/modules/updateComplete.js

```diff
@@ -0,0 +1,19 @@
+import { toDoListData } from './todo.js';
+
+export default function updateCompletedDisplay() {
+  const checkbox = document.querySelectorAll('.checkbox');
+  const label = document.querySelectorAll('.label');
+  const listItemObject = toDoListData();
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
when I check some of the tasks as completed and then refresh the page, the previously checked boxes become unchecked.

![image](https://user-images.githubusercontent.com/38088818/184485098-83a473a1-09cc-425a-8eb8-c1d9722b21e0.png)

- Kindly look into it and ensure that the boxes remain checked even after page refresh.